apps/locale/en_QQ/chromium.app

```diff
@@ -1,11 +1,12 @@
 {
-    "launch_command": "chromium-browser --start-maximized",
+    "launch_command": "kano-tracker-ctl session run 'chromium-browser' 'chromium-browser --start-maximized'",
```

Comment:
I did wonder about that why everything was different there but not under `es_AR` which was clearly the one we've worked with most. I'll get them to match, thanks.